src/test/kotlin/org/opensearch/replication/integ/rest/StartReplicationIT.kt

```diff
@@ -1255,6 +1258,62 @@ class StartReplicationIT: MultiClusterRestTestCase() {
         )
     }
 
+    fun `test operations are fetched from lucene when leader is in mixed mode`() {
```

Comment:
Minor: Can you add a opposite test case as well? Basically verify that operations are being fetched from translog if migration is not is progress.

Reply:
There is already an existing IT, which verifies the same : `test leader stats`.